.azure/linux-bench.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+- job: ghcide_bench_linux
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-latest'
+  strategy:
+    matrix:
+      stack_810:
+        STACK_YAML: "stack810.yaml"
```

Comment:
I guess there is some good reason to not use the default stack.yaml but it would be nice to use it, no?

Reply:
yeah, fair enough